packages/shared/icons/outline/x.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If we are using the filled/outlined version could we please move the existing one?
Technically there is another task for moving the SVG to the modern one.

Appreciate you already moved these, but would love to have them in the correct folders.